.vitepress/config/shared.ts

```diff
@@ -53,6 +53,11 @@ export const sharedConfig = defineConfig({
         dateStyle: "full",
       },
     },
+    editLinks: true,
+    editLinkText: 'Edit this page',
+    repo: 'oxc-project/oxc-project.github.io',
+    docsDir: 'src/docs',
+    docsBranch: 'main',
```

Comment:
I guess you have already noticed as you said, but would you write as much config (especially `editLinkText`) as possible in each locale's config? 🙏